pages/webhooks-testing.md

```diff
@@ -0,0 +1,76 @@
+# Adyen Webhooks
+
+Webhooks are HTTP callbacks sent to an endpoint on your server. These webhooks are used to receive automatic updates about the payments.
+
+This guide explains how to easily consume and test webhooks that the Adyen platform sends.
+
+
+![Webhooks testing](pages/webhooks-testing-diagram.png)
```

Comment:
@gcatanese Missing image here, will you add it?

Reply:
It is already on `main`, I pushed it my mistake